Makefile

```diff
@@ -19,7 +19,7 @@ OPEN_PR="true"
 # Force the installation of a Concrete Python version, which is very useful with nightly versions
 # /!\ WARNING /!\: This version should NEVER be a wildcard as it might create some
 # issues when trying to run it in the future.
-CONCRETE_PYTHON_VERSION="concrete-python==2024.4.19"
+CONCRETE_PYTHON_VERSION="concrete-python==2.6.2.dev20240529"
```

Comment:
btw, now that nightlies are completely public and handled directly in the lock file, I think we can safely remove this part

Reply:
That's correct, let's do that in a follow up PR

Reply:
sure yes : https://github.com/zama-ai/concrete-ml-internal/issues/4463